db/seed.py

```diff
@@ -4,9 +4,9 @@
 import os
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from models import Base, Movie, Genre, Character, Line, Conversation
+from db.models import Base, Movie, Genre, Character, Line, Conversation
 
-directory = os.path.abspath(os.path.dirname(__file__))
+directory = os.path.abspath(os.getcwd())
```

Comment:
Yes, I thought about that too. I don't know if it is possible with pipenv, but other package managers allow developers to execute scripts on hooks (postinstall, preinstall) etc. cwd would be package root in that case and allow us to stay consistent here.

Reply:
before i decided use a simple shell script, i looked into the option of creating a postinstall hook with setuptools and/or pip. maybe it's just because i'm not very familiar with the process, but it seemed very hackish and cumbersome. could creating a module level function like e.g. `db.initialize()` be a solution? for now, this is working well enough.

Reply:
`setuptools` look like best solution here. can fix in the future